internal/config/development.yaml

```diff
@@ -13,4 +13,4 @@ rabbitmq:
 
 things:
   hostname: localhost
-  port: 8082
+  port: 8182
```

Comment:
What do you think about removing `things` and `users` configuration from the `development.yaml`? In practice, we don't need them to be defined here and it just adds one more concern to us.

Reply:
The developer can define them to their local usage.

Reply:
Done!